src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -1,9 +1,10 @@
 # Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-# notice: using sets here to ensure the mapping is hashable
+# NB: You must use lowercase and replace all `-` with `_` for the requirement's name.
```

Comment:
I considered having our code normalize this, but I figure it's better to avoid the wasted computation every run.

Reply:
Ah - missed this. Every run == once per interpreter start == once for any run of Pants where you actually want to count pennies. Non pantsd is dog slow and perf considerations there aren't worth alot.

Reply:
Um, ok - once per Process. Still - I think you're prematurely asking for bugs. How about a unit test that verifies the keys are normalized?

Reply:
Great idea. Done